web/pdf_layer_viewer.js

```diff
@@ -94,7 +94,7 @@ class PDFLayerViewer extends BaseTreeViewer {
       element.textContent = this._normalizeTextContent(name);
       return;
     }
-    element.textContent = await this.l10n.get("additional_layers");
+    element.textContent = await this.l10n.get("pdfjs-additional-layers");
```

Comment:
Given that this component is only used in the default viewer, could we change this line as follows (with suitable modification of the l10n-files) and thus remove `async` from this method?
```suggestion
    element.setAttribute("data-l10n-id", "pdfjs-additional-layers");
```